evm/its.js

```diff
@@ -710,6 +659,7 @@ if (require.main === module) {
                 'setPauseStatus',
                 'execute',
                 'checks',
```

Comment:
registerTokenMetadata needs to be added, flowLimit related queries need to be changed. callContractWithInterchainToken is removed. Can you double check all entrypoints are updated correctly?

Reply:
on it